apps/web/pages/readonly-checkout.vue

```diff
@@ -55,85 +58,168 @@
 </template>
 
 <script lang="ts" setup>
-import { AddressType, type PaymentMethod, orderGetters, shippingProviderGetters } from '@plentymarkets/shop-api';
+import { AddressType, ApiError, Order, orderGetters } from '@plentymarkets/shop-api';
 import { SfLoaderCircular } from '@storefront-ui/vue';
-
-definePageMeta({
-  pageType: 'static',
-});
+import PayPalExpressButton from '~/components/PayPal/PayPalExpressButton.vue';
+import { type PayPalAddToCartCallback } from '~/components/PayPal/types';
 
 const ID_CHECKBOX = '#terms-checkbox';
-
-const { getSession } = useCustomer();
+const localePath = useLocalePath();
+const route = useRoute();
+const { send } = useNotification();
+const { t } = useI18n();
+const { isAuthorized } = useCustomer();
+const { isLoading: navigationInProgress } = useLoadingIndicator();
 const { data: cart, cartIsEmpty, clearCartItems, loading: cartLoading } = useCart();
+const { getShippingMethods } = useCartShippingMethods();
+const { data: paymentMethodData, fetchPaymentMethods, savePaymentMethod } = usePaymentMethods();
+const { loading: createOrderLoading, createOrder } = useMakeOrder();
+const { shippingPrivacyAgreement } = useAdditionalInformation();
+const { loading: executeOrderLoading, executeOrder } = usePayPal();
+const { processingOrder } = useProcessingOrder();
+const { setInitialCartTotal, changedTotal } = useShippingAsBilling();
```

Comment:
we have to load paypals order amount when accessing the page. If the customer reloads the page, the amount might not be what he confirmend in paypal. If the compare value is empty we should load the data again with getPayPalOrderDetails.